microsite/sidebars.json

```diff
@@ -243,6 +243,7 @@
     "Tutorials": [
       "tutorials/journey",
       "tutorials/quickstart-app-plugin",
+      "tutorials/configuring-plugin-databases",
```

Comment:
I think this probably belongs under the `Plugins` heading instead.

We're also working on restructuring these docs, so there's a chance that this section might be removed or moved around, so it's probably best to put it there instead.

Reply:
Actually ignore me. let's ship!